### ISSUE TITLE
[Bug and improvements] Scrollbars and few visual Improvements.

### ISSUE THREAD

**Issue by Rong-Yao** [VitalikPro13/HOLLOW#54]
### What happened?

[Feature accessibility] * The main feed doesn't have scrollbar or jump to top, jump to last red, jump to bottom.
<img width="1920" height="1040" alt="Image" src="https://github.com/user-attachments/assets/357c5793-ec38-47f9-96a5-51a76c818b58" />

[Visual bug] *  Some scrollbars on the app cover other elements instead of being on the side
<img width="1920" height="1040" alt="Image" src="https://github.com/user-attachments/assets/244366fc-4be7-4b05-ad3b-cad0a6471a34" />

[Visual nag, not a bug] * Viewing public server as a guest [message] is not styled
<img width="1920" height="1040" alt="Image" src="https://github.com/user-attachments/assets/3b341180-3040-45f4-9fe9-73837fc83e27" />

[Appearance] * Option to open user cards maximized
[Visual bug] * User cards open over the user but if you maximize the app that moves the card in the middle of the screen, instead of realigning it over the user
* Suggestion, ability to choose avatar border that will show up on the cards [not important, inspired from steam and discord]
<img width="1920" height="1040" alt="Image" src="https://github.com/user-attachments/assets/4d4807ab-ccae-41da-b712-a14ef76864ec" />

[Visual bug] * Previous messages of sticker packs show as downloading... as visible in the screenshots above
[Appearance] * Ability to collapse groups in the member list [mostly to hide offline members]
[Accessibility] * Ability to manually adjust left server panel and user member panel icon size and text size, ye a friend mentioned that it would be nice if he could adjust those as well.

### Steps to reproduce

For the card, click on a user
Maximize the app
The card remains centered on the screen covering the chat instead

For the scrollbars
Click on labels, manage labels, if your interface is zoomed in you will noticably see the scrollbar covers the user checkboxes on a side.

There is no easy way to scroll far up or far down in the feed.

### Expected behavior

The cars need to recenter over the user if app is maximized i think

the scrollbars in the label management are not supposed to cover checkboxes

there should be an easier way to scroll up and down the chat to find a message or have some sort of search

The notifications panel on top of the app for news and you viewing as a guest, both should follow the theme of the app

### Hollow version

0.9.4

### Operating system

Windows 10

### Relay

Default (relay.anonlisten.com)

### Logs (optional)

```text

```

**Comment by NhoxLeggy** [comment]
Point 1 : Yeah, it's really missing a scrollbar for the chat to navigate faster when you have a ton of messages.
Point 2 : per example, the scrollbar in the settings are on top of the text boxes : 
<img width="485" height="434" alt="Image" src="https://github.com/user-attachments/assets/1b48d286-3147-44b3-a2ed-0a74e6803f5d" />
I couldn't reproduce point 5 from my end, otherwise I agree regarding everything else, especially being able to resize the left panel (with the channel's servers).

**Comment by VitalikPro13** [comment]
Understood almost all the stuff. Regarding "Option to open user cards maximized," do you mean it should open the full profile with the showcase by default with one click, instead of the little icon in the top-right corner of the banner? I can add a toggle for that and fix all the rest too. I'm a bit busy implementing optimizations to support more viewers in screen sharing instead of having the sharer stream all the bandwidth and such. I'll try to get it done quickly, and then do this issue.

**Comment by Rong-Yao** [comment]
To be able to choose if on left click on a profile you see the profile card maximized or small like in the two screenshots below.

<img width="1920" height="1040" alt="Image" src="https://github.com/user-attachments/assets/3adf104c-abf3-46eb-98c5-4fb7eb508566" />

<img width="1920" height="1040" alt="Image" src="https://github.com/user-attachments/assets/4e5d4fa8-f5ca-4a0f-a157-b0cda6858193" />

**Comment by VitalikPro13** [comment]
Hey! A little update on the current stuff. I haven't started on the bug fixes with improvements on this yet, BUT I almost finished with the screen share forwarder (the pre-last step is done, with testing left to do). When it's done, then I can fix all these issues. Though, I'll be leaving for the hospital appointment to treat my skin condition, so I won't be able to make new commits for a week. Honestly, feels like I need this break from my daily development grind... yeah, better do it because I definitely don't want to burn out or something. Thank you for all the support and for contributing to the project by creating Issues for bugs or suggestions! You can't believe how much I appreciate it!

**Comment by VitalikPro13** [comment]
Everything is implemented in v0.10.1, including the avatar frames (+ animations). Since Hollow avatars are rounded squares, you can literally download some frames from the Steam points shop and use them! Full aPNG format support, and it works for everybody. The rest is implemented too. Just look at this:

https://github.com/user-attachments/assets/cc8f9edc-add9-4d9f-9553-e4fd38c06a12
